build-system/tasks/check-links.js

```diff
@@ -123,6 +134,7 @@ function runLinkChecker(markdownFile) {
   }
   var markdown = fs.readFileSync(markdownFile).toString();
   var filteredMarkdown = filterLocalhostLinks(markdown);
+  filteredMarkdown = filterScriptTagLinks(filteredMarkdown);
```

Comment:
Would it make sense to filter all links that start with https? and don't have a https://github.com/ampproject/ prefix?